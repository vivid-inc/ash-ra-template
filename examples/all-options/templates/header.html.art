{%
(require '[hiccup.core :as hiccup])

(def main-items ["Solutions" "/solutions/"
                 "Services" "/services/special.html"
                 "Support" "/wide/support/"])

%}
<!-- Updated {%= updated %} -->
<nav>
  {%= (hiccup/html
        (for [[title href] (partition 2 main-items)]
          [:ul.main-item [:a {:href href} title]])) %}
</nav>
