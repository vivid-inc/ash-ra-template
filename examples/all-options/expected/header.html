
<!-- Updated 2021-01-01 -->
<nav>
  <ul class="main-item" href="/solutions/">Solutions</ul><ul class="main-item" href="/services/special.html">Services</ul><ul class="main-item" href="/wide/support/">Support</ul>
</nav>
